pyaedt/common_rpc.py

```diff
@@ -120,7 +120,9 @@
 
 
 def pyaedt_service_manager(port=17878, aedt_version=None, student_version=False):
-    """Starts an RPyC server on CPython and listens on a specified port.
+    """Starts PyAEDT service manager using RPyC server on CPython and listens on a specified port.
```

Comment:
```suggestion
    """Starts the PyAEDT service manager using RPyC server on CPython.
```
I think the last portion of this sentence can be deleted since the additional detail that follows it talks about how it listens on a dedicated port.